### PR TITLE
chore: Backport of marktext#2414

### PR DESCRIPTION
**Pull request by fxha**
> Add option to disable HTML rendering

Backport of https://github.com/marktext/marktext/pull/2414.

Affected by https://github.com/marktext/marktext/issues/2982.

### CODE REVIEW

File: lib/utils/index.js [hunk 1]
```diff
@@ -214,10 +222,16 @@ export const generateKeyHash = keys => {
 // mixins
 export const mixins = (constructor, ...object) => Object.assign(constructor.prototype, ...object)
 
-export const sanitize = (html, options) => {
-  const DOMPurify = createDOMPurify(window)
+export const escapeHtmlTags = html => {
```
Comment (Jocs) [inline lib/utils/index.js:225]: is `escapeHtmlTags` same to `escapeHTML` ?

**Comment by Jocs** [comment]
@fxha If we disable HTML rendering, can we render the `html block` like `code block`, by making the edit area always show and hiding the preview of the html?

**Comment by fxha** [comment]
> If we disable HTML rendering, can we render the html block like code block, by making the edit area always show and hiding the preview of the html?

I guess this would be an improvement. Feel free to implement it and create a PR for MarkText too.

**Comment by fxha** [comment]
@Jocs Do you think there is a huge benefit from using commitlint? Forcing a prefix like `chore` when squash merging feels right, but not for each commit (especially during development).

**Comment by Jocs** [comment]
Of course it's best to verify through `commitlint` when we merge into master. and not each commit.

**Pull request merged**